test/python/WMCore_t/Services_t/Rucio_t/Rucio_t.py

```diff
@@ -237,7 +237,26 @@ def testGetPFN(self):
         """
         Test `getPFN` method
         """
-        self.assertRaises(NotImplementedError, self.myRucio.getPFN)
+        # simplest call
+        resp = self.myRucio.getPFN(site="T2_CH_CERN", lfns="/store")
```

Comment:
Stefano, quick comment here. This Rucio object is connected to the Rucio integration server, which seems to be unable to resolve LFN x PFN for the main RSE names. I don't know the reason neither whether it's expected or not, however, could you please update your RSE names to actually their `_Test` correspondents? Thanks

Reply:
thanks Alan, but...
- I found in test log: getPFN() takes at least 3 arguments (1 given) 
   so maybe something else is going on (too)
- this lnf2pfn resolution fails a lot also in main server and I pissed off.
- I do not know how to use test server so that I can first test things in my python shell, and I have no confidence that "all _Test" sites will be there. For testing protocol (scheme in Rucio) only a few sites can do.